packages/slice-machine/src/legacy/components/Navigation/index.tsx

```diff
@@ -44,6 +46,9 @@ const Navigation: FC = () => {
   const { repositoryName, repositoryDomain, repositoryUrl } =
     useRepositoryInformation();
   const gitIntegrationExperiment = useGitIntegrationExperiment();
+  const [isSliceLibraryDialogOpen, setIsSliceLibraryDialogOpen] =
+    useState<boolean>(false);
```

Comment:
I think you don't need that 👍

```suggestion
    useState(false);
```

Reply:
This was what I tried at first, but I could swear it gave me a type error — I'm not getting one now, so I'm not sure what that was ~~" It certainly makes more sense for it to be inferred, if it can be.

Reply:
Changed